web/src/components/apps/AppDetailPage.tsx

```diff
@@ -520,7 +520,7 @@ function AppDetailPage(props: Props) {
                   render={() => (
                     <TroubleshootContainer
                       app={selectedApp}
-                      appName={appName}
+                      appName={appName || ""}
```

Comment:
think the typechecks script is not returning an error on failure. will investigate in another pr. This was a type error i saw while scrolling through the component